app/helpers/application_helper.rb

```diff
@@ -7,6 +7,17 @@ module ApplicationHelper
   include StiRoutingHelper
   include TextualSummaryHelper
 
+  def icon_text(i, text = "", opts = {})
```

Comment:
Something like `icon_text(kind, i, ...` with the class name being generated as `"#{kind} #{kind}-#{i}"` would work. (Even though `.product` and `.pficon` don't exist by themselves.)